data/dapps/malda.json

```diff
@@ -0,0 +1,6 @@
+{
+  "aliases": {
+    "malda": { "chains": ["linea"], "title": "Malda" }
```

Comment:
There is no Defillama page, but they use `Malda` consistently in their webiste, logo and twitter.